template/template_test.go

```diff
@@ -113,6 +113,42 @@ func TestEmptyValueWithHardDefault(t *testing.T) {
 	assert.Check(t, is.Equal("ok ", result))
 }
 
+func TestPrensentValueWithUnset(t *testing.T) {
```

Comment:
FYI you've got a typo on "Present" here (`Prensent`) which is copy-pasted to the other test cases too 😛

Reply:
Ops! My bad! 
Good catch!